src/views/ConfigEditor.tsx

```diff
@@ -77,6 +77,24 @@ export class ConfigEditor extends PureComponent<ConfigEditorProps> {
             />
           </div>
         </div>
+        <div className="gf-form-group">
+          <h3 className="page-heading">Github Enterprise</h3>
+          <div className="gf-form">
+            <InlineFormLabel className="width-10">Github Enterprise URL</InlineFormLabel>
+            <Input
+              css=""
+              className="width-27"
+              value={jsonData.githubUrl}
+              placeholder="URL of Github Enterprise"
+              summary="URL for Github Enterprise, such as https://github.company.com, leave blank if using github.com"
+              onChange={onUpdateDatasourceJsonDataOption(this.props, 'githubUrl')}
+            />
+          </div>
+        </div>
+        <InfoBox title="Github Enterprise">
+          <p>For Github Enterprise enter the URL, such as https://github.company.com</p>
+          <p>Leave blank if not using Github Enterprise, which will default to github.com</p>
```

Comment:
```suggestion
          <p>For GitHub Enterprise enter the URL, such as https://github.company.com</p>
          <p>Leave blank if not using GitHub Enterprise, which will default to github.com</p>
```

Reply:
Fixed all of these to "GitHub", but I noticed that when they were displayed in the form, it reverted to "Github".

Reply:
I tried a new browser too thinking it was a browser cache issue, but same issue :(

Reply:
hmm. well as long as the text in here is fine then I think it's ok

Reply:
It could be that you needed to run `yarn dev`